src/Components/Subscriptions/Table/index.jsx

```diff
@@ -136,12 +143,20 @@ const Table = ({ data }) => {
         </div>
       )}
       {showConfirmDeleteModal && (
-        <ConfirmDeleteModal
-          onClose={() => setShowConfirmDeleteModal(false)}
-          confirmDelete={() => handleDelete(editingSubscriptionId)}
-        />
+        <ConfirmModal
+          title="Delete Subscription"
+          handler={closeModal}
+          onAction={() => handleDelete(editingSubscriptionId)}
+          reason="delete"
+        >
+          <p>Are you sure to delete subscription?</p>
+        </ConfirmModal>
+      )}
+      {showDeleteModal ? (
+        <ResponseModal handler={closeModal} state="fail" message="Subscription Deleted" />
```

Comment:
I think if you are saying the subscription was deleted, it's not a state of "fail", because was successfully deleted. I know that the color maybe is more appropriate on fail state, but I think it's need to be success instead.